diff --git "a/Telegram/SourceFiles/data/components/sponsored_messages.cpp" "b/Telegram/SourceFiles/data/components/sponsored_messages.cpp"
index b907f1d..74af3e7 100644
--- "a/Telegram/SourceFiles/data/components/sponsored_messages.cpp"
+++ "b/Telegram/SourceFiles/data/components/sponsored_messages.cpp"
@@ -230,12 +230,7 @@ void SponsoredMessages::inject(
 	}
 }
 
-bool SponsoredMessages::canHaveFor(not_null<History*> history) const {
-	if (history->peer->isChannel()) {
-		return true;
-	} else if (const auto user = history->peer->asUser()) {
-		return user->isBot();
-	}
+bool SponsoredMessages::canHaveFor(not_null<History*> history) const {
 	return false;
 }
 
@@ -245,11 +240,6 @@ bool SponsoredMessages::canHaveFor(not_null<HistoryItem*> item) const {
 }
 
 bool SponsoredMessages::isTopBarFor(not_null<History*> history) const {
-	if (peerIsUser(history->peer->id)) {
-		if (const auto user = history->peer->asUser()) {
-			return user->isBot();
-		}
-	}
 	return false;
 }
 
diff --git "a/Telegram/SourceFiles/data/data_peer.cpp" "b/Telegram/SourceFiles/data/data_peer.cpp"
index cbc27b5..a0d8e76 100644
--- "a/Telegram/SourceFiles/data/data_peer.cpp"
+++ "b/Telegram/SourceFiles/data/data_peer.cpp"
@@ -1759,11 +1759,6 @@ const Data::WallPaper *PeerData::wallPaper() const {
 }
 
 bool PeerData::hasActiveStories() const {
-	if (const auto user = asUser()) {
-		return user->hasActiveStories();
-	} else if (const auto channel = asChannel()) {
-		return channel->hasActiveStories();
-	}
 	return false;
 }
 
diff --git "a/Telegram/SourceFiles/dialogs/dialogs_row.cpp" "b/Telegram/SourceFiles/dialogs/dialogs_row.cpp"
index 1082a20..a4afed9 100644
--- "a/Telegram/SourceFiles/dialogs/dialogs_row.cpp"
+++ "b/Telegram/SourceFiles/dialogs/dialogs_row.cpp"
@@ -421,7 +421,7 @@ void Row::PaintCornerBadgeFrame(
 	q.translate(framePadding, framePadding);
 	auto hq = std::optional<PainterHighQualityEnabler>();
 	const auto photoSize = context.st->photoSize;
-	const auto storiesCount = data->storiesCount;
+	const auto storiesCount = 0;
 	if (storiesCount) {
 		hq.emplace(q);
 		const auto line = st::dialogsStoriesFull.lineTwice / 2.;
diff --git "a/Telegram/SourceFiles/dialogs/dialogs_widget.cpp" "b/Telegram/SourceFiles/dialogs/dialogs_widget.cpp"
index ffcd9dc..b6ee0bf 100644
--- "a/Telegram/SourceFiles/dialogs/dialogs_widget.cpp"
+++ "b/Telegram/SourceFiles/dialogs/dialogs_widget.cpp"
@@ -363,12 +363,7 @@ Widget::Widget(
 	object_ptr<Ui::IconButton>(this, st::dialogsLock))
 , _scroll(this)
 , _scrollToTop(_scroll, st::dialogsToUp)
-, _stories((_layout != Layout::Child)
-	? std::make_unique<Stories::List>(
-		this,
-		st::dialogsStoriesList,
-		_storiesContents.events() | rpl::flatten_latest())
-	: nullptr)
+, _stories(nullptr)
 , _searchTimer([=] { search(); })
 , _peerSearch(&controller->session(), Api::PeerSearch::Type::WithSponsored)
 , _singleMessageSearch(&controller->session()) {
diff --git "a/Telegram/SourceFiles/info/profile/info_profile_widget.cpp" "b/Telegram/SourceFiles/info/profile/info_profile_widget.cpp"
index a11face..d63c688 100644
--- "a/Telegram/SourceFiles/info/profile/info_profile_widget.cpp"
+++ "b/Telegram/SourceFiles/info/profile/info_profile_widget.cpp"
@@ -132,10 +132,6 @@ rpl::producer<QString> Widget::title() {
 }
 
 rpl::producer<Dialogs::Stories::Content> Widget::titleStories() {
-	const auto peer = controller()->key().peer();
-	if (peer && !peer->isChat()) {
-		return Dialogs::Stories::LastForPeer(peer);
-	}
 	return nullptr;
 }
 
